gatsby-node.js

```diff
@@ -27,25 +27,26 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     return
   }
 
-  const posts = result.data.allContentfulBlogPost.nodes
+  const caseStudies = result.data.allContentfulCaseStudy.nodes
 
-  // Create blog posts pages
-  // But only if there's at least one blog post found in Contentful
+  // Create case study pages
+  // But only if there's at least one case study found in Contentful
   // `context` is available in the template as a prop and as a variable in GraphQL
 
-  if (posts.length > 0) {
-    posts.forEach((post, index) => {
-      const previousPostSlug = index === 0 ? null : posts[index - 1].slug
-      const nextPostSlug =
-        index === posts.length - 1 ? null : posts[index + 1].slug
+  if (caseStudies.length > 0) {
+    caseStudies.forEach((caseStudy, index) => {
+      const previousCaseStudySlug =
+        index === 0 ? null : caseStudies[index - 1].slug
+      const nextCaseStudySlug =
+        index === caseStudies.length - 1 ? null : caseStudies[index + 1].slug
 
       createPage({
-        path: `/blog/${post.slug}/`,
+        path: `/caseStudy/${caseStudy.slug}/`,
         component: blogPost,
```

Comment:
This should be case study not blog

Reply:
Or is it using blogpost component for now?

Feel free to raise this on a separate ticket if you don’t think it’s part of this PR

Reply:
Mand I have fixed it. I have also fixed the template so it is working now. Next steps would be looking at previous and next buttons. Still have to figure that out why it's not working.